### PR TITLE
Document how to enable DEBUG logging for @QuarkusTest

### LINKED ISSUE #6831: Document how to enable DEBUG logging for @QuarkusTest
I've tried to set system properties in `applications.properties` and for the `maven-surefire-plugin` with no success.

### PR DESCRIPTION
**Pull request by mkouba**
- resolves #6831

### CODE REVIEW

**Review by geoand — APPROVED** [review]
Good tip!

File: docs/src/main/asciidoc/logging.adoc [hunk 1]
```diff
@@ -194,10 +194,36 @@ Applications and components may use any of the following APIs for logging, and t
...
 * https://commons.apache.org/proper/commons-logging/[Apache Commons Logging]
 
-== Centralized log management
+== Centralized Log Management
 
 If you want to send your logs to a centralized tool like Graylog, Logstash or Fluentd, you can follow the link:centralized-log-management[Centralized log management guide].
 
+== How to Configure Logging for `@QuarkusTest`
+
+If you want to configure logging for you `@QuarkusTest` don't forget to setup the `maven-surefire-plugin` accordingly.
```
Comment (gsmet) [inline docs/src/main/asciidoc/logging.adoc]:
```suggestion
If you want to configure logging for your `@QuarkusTest`, don't forget to set up the `maven-surefire-plugin` accordingly.
```

**Review by gsmet — CHANGES_REQUESTED** [review]
Spotted one typo and fixed a few other things while at it :).

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
Merged, thanks!

**Pull request merged**